recipes/findorb/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "findorb" %}
+{% set version = "0.0.20210525" %}
```

Comment:
Have you tried asking upstream if they'd be prepared to start adding tags now and again?

Reply:
It's being discussed, but I'm not sure how long it may take -- it's not very common very in this community. A similar discussion on [openorb](https://github.com/conda-forge/openorb-feedstock/) has taken ~2 years.